cab-token-generator/java/com/google/auth/credentialaccessboundary/ClientSideCredentialAccessBoundaryFactory.java

```diff
@@ -207,7 +207,7 @@ public AccessToken generateToken(CredentialAccessBoundary accessBoundary)
     byte[] encryptedRestrictions = this.encryptRestrictions(rawRestrictions, sessionKey);
 
     String tokenValue =
-        intermediateToken + "." + Base64.getUrlEncoder().encodeToString(encryptedRestrictions);
+        intermediateToken + "." + Base64.getUrlEncoder().withoutPadding().encodeToString(encryptedRestrictions);
```

Comment:
Can you please update the relevant unit tests to reflect this change, and add a comment here explaining why we're using `withoutPadding()`.

Reply:
Added tests.

Reply:
Thanks! Can you also add a comment here documenting why there shouldn't be any padding?

Reply:
Added.

Reply:
nit: Can you add a bit more to explain the `why`? Is there a specific behavior/requirement from server-side CAB that needs `withoutPadding` or just something that was always done? If not, can this comment be added like `withoutPadding() avoids URL encoded token issues (i.e. `=` in the path)`, etc.

Reply:
Done.